### PR TITLE
Added contact form and footer.

### PR DESCRIPTION
**Pull request by PabloEGonz**
In this branch, I added a contact form with required fields, and the data is sent to Formspree, I also added the footer with icons that link to my profiles.

### CODE REVIEW

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @PabloEGonz and @stino-x 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Create forms with HTML5 validation ✔️
Collect data by using the [Formspree service](https://formspree.io/html). ✔️
Flexbox is used to layout the form elements ✔️ 
Implement button interactions ✔️ 

## Required Changes ♻️

- [ ] Please kindly fix the two broken links in the **client** section of your Readme file, I got 404 error. Implementing this will make your repo more engaging. Thanks. 🙏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -216,5 +216,72 @@ <h2>About me</h2>
         </ul>
       </div>
     </section>
+    <section id="last">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/xwkjrpwp" method="post">
```
[the remark this answers is not in the document]
Comment (PabloEGonz) [reply index.html:224]: hello, @OyePriscilla I don't quite understand the suggestions here, I made sure the typography is the same, can you please clarify?

[the remark this answers is not in the document]
Comment (PabloEGonz) [reply index.html:224]: @OyePriscilla thanks for the clarification, I'll make sure to implement it.

File: style.css [hunk 2]
```diff
@@ -405,3 +426,101 @@ li {
...
+  width: 90%;
+  font-family: "Inter", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 20px 0 20px 15px;
+  background: #fbf8f7;
```
Comment (binhussen) [inline style.css:477]: - Kindly, the background color for the form input field isn't necessary, as the previous code reviewer mentioned, please remove it to have the same look as the given [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-27&t=qfQFTtA4uea6IzVw-0) and better UI :+1:

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @PabloEGonz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**